srml/council/src/seats.rs

```diff
@@ -429,7 +437,7 @@ impl<T: Trait> Module<T> {
 	}
 
 	// Private
-	/// Check there's nothing to do this block
+	/// Check that there's nothing to do this block.
```

Comment:
suggest change to:

```
  /// Checks to perform at the end of each block.
  ///
  /// - Check if the voting period ends, and if there are any empty council seats start the presentation period (until the finalization period starts), otherwise start a new vote.
  /// - Check if the finalization period starts.
```